pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv
+, fetchurl
+, intltool
+, bash
+, which
+, parallel
+, pkgconfig
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gnome3
+, gtk-engine-murrine
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "3.1.2";
+
+  src = fetchurl {
+    url    = "https://github.com/pop-os/gtk-theme/archive/${version}.tar.gz";
+    sha256 = "15pj5rhhpn3clb87973m9bbmp02z4rv0d88h0idmpdg4i0yj5q8a";
+  };
+
+  nativeBuildInputs = [
+    bash
+    which
+    parallel
+    intltool
+    sassc
+    gtk3
+    inkscape
+    optipng
+  ];
+
+  propagatedUserEnvPkgs = [
+    gnome3.gnome-themes-extra
+    gtk-engine-murrine
+  ];
+
+  preBuild = ''
+    export HOME="$NIX_BUILD_ROOT"
+
+    patchShebangs .
+
+    substituteInPlace Makefile \
+      --replace '/bin/bash' '${bash}/bin/bash'
+
+    substituteInPlace Makefile \
+      --replace '/usr/share/themes' '${placeholder "out"}/share/themes'
+
+    substituteInPlace src/gtk-2.0/render-asset.sh \
+      --replace '/usr/bin/inkscape' '${inkscape}/bin/inkscape'
+
+    substituteInPlace src/gtk-2.0/render-asset.sh \
+      --replace '/usr/bin/optipng' '${optipng}/bin/optipng'
+
+    substituteInPlace src/gtk-2.0/render-assets-dark.sh \
+      --replace '/usr/bin/inkscape' '${inkscape}/bin/inkscape'
+
+    substituteInPlace src/gtk-2.0/render-assets-dark.sh \
+      --replace '/usr/bin/optipng' '${optipng}/bin/optipng'
+
+    substituteInPlace src/gtk-3.0/gtk-common/render-asset.sh \
+      --replace '/usr/bin/inkscape' '${inkscape}/bin/inkscape'
+
+    substituteInPlace src/gtk-3.0/gtk-common/render-asset.sh \
+      --replace '/usr/bin/optipng' '${optipng}/bin/optipng'
```

Comment:
You could simplify this a bit with a loop like so:

```
    for file in $(find -name render-asset\*.sh); do
      substituteInPlace "$file" \
        --replace 'INKSCAPE="/usr/bin/inkscape"' \
                  'INKSCAPE="${inkscape}/bin/inkscape"' \
        --replace 'OPTIPNG="/usr/bin/optipng"' \
                  'OPTIPNG="${optipng}/bin/optipng"'
    done
```

Reply:
Could we PR upstream just not using `/usr/bin/` at all?
They could just expect it to be in PATH, it makes more sense. Though it could be better even.

Reply:
This release is more than one year old. Upstream current development changed to the meson build system.

Reply:
This release is more than one year old. Recently upstream development seems to have been changed to the meson build system.

Reply:
And, it's possible the theme is completely broken with gnome 3.34?
If I understand it correctly, that release probably targeted whatever was in ubuntu bionic.
So we might have to use the HEAD of `master_eoan` branch.

Reply:
Uugh, they made the Makefile parametric in 4.0b2 but the meson port uses hardcoded paths in the scripts again.

Reply:
> And it's possible the theme is completely broken with gnome 3.34?
> If I understand it correctly, that release probably targeted whatever was in ubuntu bionic.
> So we might have to use the HEAD of `master_eoan` branch.

I have started to work on supporting the v4, to anticipate the release but I submitted the 3.1.2 because it was the latest stable. Do you want me to submit the v4 directly even if it's a beta ?

Reply:
It looks like that System76-Pop! OS distributes [a version based on HEAD](https://launchpad.net/~system76/+archive/ubuntu/pop/+sourcepub/10732034/+listing-archive-extra) of `master_eon`. Probably we should do the same.

Reply:
Sounds good @romildo as it looks we agreed. @Elyhaka Could you do this?

Reply:
Of course, working on it !

Reply:
Done, I have been able to remove some dependencies in the meantime.

Reply:
> Of course, working on it !

Oh sorry :smile:, I just wanted to ask properly because we have tendencies to just **suggest** instead stating "hey, this?

Reply:
Oh, I see ! I didn't took it like that, no problem :)